internal/manifests/manifests.go

```diff
@@ -450,6 +477,24 @@ func (m *Manifests) validateUserSuppliedManifest(ctx context.Context, clusterID
 	return nil
 }
 
+// isValidYaml checks if one or more (in case of multi-doc yaml) yaml documents are valid
+func isValidYaml(manifestContent []byte) bool {
+	dec := yaml.NewDecoder(bytes.NewReader(manifestContent))
+
+	for {
+		var doc interface{}
+		err := dec.Decode(&doc)
+		if errors.Is(err, io.EOF) {
+			break
+		}
+		if err != nil {
+			return false
```

Comment:
are we not interested in what the error was, should we return it, should we log it?

Reply:
We were not handling the error previously, I did not change that.

I don't think we should log anything here, it's expected that sometimes users will upload invalid yamls.

wdyt?

Reply:
I think @paul-maidment is right, we should return the error and add it when logging the error

Reply:
I now return the error to the user